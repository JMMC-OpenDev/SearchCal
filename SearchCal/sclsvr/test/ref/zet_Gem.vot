version https://git-lfs.github.com/spec/v1
oid sha256:93500188dc9a450c37ba50e57b96be069d2326d96059acf551fec00a3bb58991
size 160941
