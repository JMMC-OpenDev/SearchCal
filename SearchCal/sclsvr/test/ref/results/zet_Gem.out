version https://git-lfs.github.com/spec/v1
oid sha256:21d2a3de23fe31756b81d0acde3351cfa84bf72cd39d72977aec6618c12eec31
size 108101
